src/components/Adyen/Adyen.js

```diff
@@ -240,7 +240,7 @@ const Adyen = ({
         return onSubmit(state, component);
       },
       onActionHandled: () => {
-        if (type === 'bank') {
+        if (type === 'zeroPaymentNotSupported') {
```

Comment:
nice to have: we could move zeroPaymentNotSupported and zeroPaymentSupported to a cont to avoid typos
const ZEROPAYMENTSUPPORTED = 'zeroPaymentSupported';
const ZEROPAYMENTNOTSUPPORTED = 'zeroPaymentNotSupported';

Reply:
or as we are using 'bank' and 'standard' in code we could do: 
const STANDARD_PAYMENT_METHODS = 'zeroPaymentSupported';
const BANK_PAYMENT_METHODS = 'zeroPaymentNotSupported';